src/library_gl.js

```diff
@@ -4,7 +4,14 @@
  */
 
 var LibraryGL = {
+#if USE_PTHREADS
+  // If GLctxIsOnParentThread is true, then
```

Comment:
`GLctxIsOnParentThread` is more self-explanatory to suggest the type is a boolean, whereas `proxiedGLctx` could be mistaken with `GLctx` to be some kind of object, but just a proxied flavor, instead of the real thing. I'll change to `GLctxIsProxied` to hint at the boolean nature?

Reply:
Sounds good.